### PR TITLE
Added Stink Bugs Strategic Plan page

### PR DESCRIPTION
**Pull request by weihuang12**

### CODE REVIEW

File: _data/community.yml
Comment (StevenCannon-USDA) [inline _data/community.yml:1]: Is a Soybean Breeder's Workshop page being developed separately? I see the new `url: "/community/sbw"` in this file, but there is no corresponding directory or index page.

**Review by StevenCannon-USDA — COMMENTED** [review]
The new page looks OK to me, but I have a question about the community/sbw link (in data/community.yml), and I have a general question about how to handle submodule updates. This is probably a question for @nathanweeks, and for discussion in the Thursday afternoon meeting.

The submodule question: when we see modifications in a branch like this, should the submodules be updated as part of the commit? If so, how?
```
git status
On branch sbsp
Your branch is up to date with 'origin/sbsp'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   ../_data/datastore-metadata (new commits)
	modified:   ../_themes/jekyll-theme-legumeinfo (new commits)
```

File: _data/community.yml
Comment (weihuang12) [reply _data/community.yml:1]: Yes. I'm still working on the Soybean Breeder Workshop page which is on a different branch sbw.

Comment (StevenCannon-USDA) [reply _data/community.yml:1]: OK, I thought maybe that was the case. That makes sense. I'll "approve" - and we can also discuss at the afternoon meeting.

Comment (weihuang12) [reply _data/community.yml:1]: I have updated both sbsp and sbw urls in the community.yml fiie, but I'm still working on the index.html for 
/community/sbw. Sorry for the confusion.

**Review by StevenCannon-USDA — DISMISSED** [review]

**Comment by StevenCannon-USDA** [comment]
@nathanweeks -
I think we still need your help with the theme submodule. I have tried several things, but they all still end with a "modified" status for jekyll-theme-legumeinfo:
```
git submodule init
git submodule update --recursive
git status
On branch sbsp
Your branch is up to date with 'origin/sbsp'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   _themes/jekyll-theme-legumeinfo (modified content, untracked content)
```

**Comment by nathanweeks** [comment]
Looks like there are modified & additional files in your local _themes/jekyll-theme-legumeinfo ? You can get a listing of what's different by setting your working directory to the submodule directory & issuing `git status`; e.g.:

    cd _themes/jekyll-theme-legumeinfo && git status

Or, an unnecessary but handy shortcut I just found out about to issue `git status` in all of your submodule directories:

    git submodule foreach --recursive git status

**Comment by StevenCannon-USDA** [comment]
@nathanweeks - I ended up brute-forcing it. That is: I stashed the changes, re-cloned the repository, started a new branch off main (sbsp2), and applied the changes. I will open a new PR for it.

**Pull request closed without merging**